attributes/defaults.rb

```diff
@@ -18,17 +18,23 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-# Attribute that defines whether MariaDB or MySQL should be used
-default['mysqld']['use_mariadb'] = true
+# Attribute that defines whether MariaDB, MySQL or Percona should be used
+default['mysqld']['db_install'] = 'mariadb'
```

Comment:
Ah i understand. Vor MariaDB, this is solved as the default recipe includes the apt_repository recipe, too. So, while you can use the cookbook without the apt repository, the default "just works". For percona, you're right. One option would be to just leave it as it is (and add a warning to the README), or automatically include the apt_repository in the install recipe when percona is selected in the attribute.